src/ngx_http_lua_shdict.c

```diff
@@ -2983,6 +2983,35 @@ ngx_http_lua_ffi_shdict_set_expire(ngx_shm_zone_t *zone, u_char *key,
 
     return NGX_OK;
 }
+
+
+size_t
+ngx_http_lua_ffi_shdict_capacity(ngx_shm_zone_t *zone)
+{
+    return zone->shm.size;
+}
+
+
+#    if nginx_version >= 1011007
+size_t
+ngx_http_lua_ffi_shdict_free_space(ngx_shm_zone_t *zone)
+{
+    ngx_http_lua_shdict_ctx_t   *ctx;
+    size_t                       free_page_bytes;
```

Comment:
For such a short-lived local auto variable, we'd better use a short name like `bytes`?